arbeitszeit_flask/templates/macros/plan_details.html

```diff
@@ -58,6 +62,9 @@ <h1 class="title">
                 <div class="tile is-4">
                     <div class="tile is-parent">
                         <article class="tile is-child notification has-background-warning-light">
+                            <div class="has-text-right has-text-info-dark">
+                                {{ info_tooltip(id="id_plan_status", text=gettext("Status can be \"Active\", i.e. plan was approved and current date is within planning timeframe, or \"Inactive\", i.e. plan is not yet approved or it is expired.")) }}
```

Comment:
can plan be approved but be Inactive and not expired because current date is BEFORE start of planning timeframe?

Reply:
No, currently not.